src/App.tsx

```diff
@@ -1,13 +1,34 @@
 import './App.css';
 import WishList from './components/WishList';
 import Form from './components/Form'; 
+import { useState } from 'react';
+
+type Item = {
+  id?: number,
+  name: string, 
+  price: number,
+  link: string,
+  image: string
+}
 
 const App = () => {
+  const [items, setItems] = useState<Item[]>([])
+
+  const addItem = (data: Item) => {
```

Comment:
```suggestion
  const addItem = ({ name, price, link, image }: Item) => {}
```
data展開してpropsで受け取った方がわかりやすいかなと思います:bow:
`name: data.name,`を`name,`に省略できるので。

Reply:
data展開ですね！すみませんでした、昨日と同じことですね🙏🏻修正します！